### DISCUSSION TITLE
Hosting coolify on an x86 VPS

### DISCUSSION THREAD

**Discussion by mikekok** [coollabsio/coolify#4443] [Q&A] [answered]
I'm thinking about switching to Coolify for my projects and wanted to know if netcup's x86 VPS would be a good choice (their ARM64 VPS are all sold out).

The [VPS 1000 G11](https://www.netcup.com/en/server/vps) option (4 cores, 8 GB RAM, 256 GB SSD) looks like a solid choice, but I would like to know if Coolify would run fine on an x86 VPS before I purchase anything (I'm no server expert as you can tell...)

I currently want to host Coolify along with three static websites, three pocketbase instances and two low traffic sveltekit projects on the same server.

**Comment by peaklabs-dev** [comment, marked as answer]
This should work just fine, as long as your server CPU architecture is one of these, it will work https://coolify.io/docs/installation#supported-architectures (you server should work). Also, you do not need that much resources to start, you can buy a smaller server and scale up as your resource usage increases.

**Reply by mikekok** [reply]
Thanks for clarifying, I'll go with this one (I'm about to add more stuff pretty soon so that's why I'm going for the bigger one).
